### PR TITLE
fix: Including a template for PRs and Issues

### PR DESCRIPTION
**Pull request by shortdiv**
Including a template for PRs and Issues for consistency and legibility.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<!--- What problem does this change solve? -->
+<!--- Include an issue number, if the issue this PR solves has been logged previously  -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds new functionality)
+- [ ] Breaking change (fix/feature that changes existing functionality)
+
+## PR Checklist:
```
Comment (freakboy3742) [inline .github/pull_request_template.md]: This is a nifty idea; might be nice to add a "all new features have been documented" option here, too.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Context
+<!--- What problem does this change solve? -->
+<!--- Include an issue number, if the issue this PR solves has been logged previously  -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds new functionality)
+- [ ] Breaking change (fix/feature that changes existing functionality)
```
Comment (freakboy3742) [inline .github/pull_request_template.md]: "Cosmetic" - no functionality; just cleaning up style or typos.

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<!--- If you're describing a bug, tell us what you expect to happen. -->
+
+## Current Behavior
+<!--- If you're describing a bug, what currently happens? -->
+
+## Steps to reproduce
+<!--- Provide a set of steps describing how to reproduce this bug. If you have a live example, provide the link below -->
+
+## Your Environment
+<!--- Provide details on your current environment you found the bug in -->
```
Comment (freakboy3742) [inline .github/issue_template.md]: Might be helpful to provide some pro-forma checkboxes here, too, so we make sure we capture Python versions, OS versions, and so on.

File: .github/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
```
Comment (phildini) [inline .github/issue_template.md]: It would be amazing if we could add some instructions or ideas for what to do if it's a feature request.

Comment (shortdiv) [inline .github/issue_template.md]: Yeah! Do you have suggestions on what to ask/how that should be formatted?

Comment (phildini) [inline .github/issue_template.md]: It's pretty conditional, sadly.

Maybe something like "Why would you like this feature?"

File: .github/issue_template.md (hunk 3 above)
Comment (shortdiv) [inline .github/issue_template.md]: Is there a list of supported versions?

File: .github/issue_template.md (hunk 4 above)
Comment (shortdiv) [inline .github/issue_template.md]: Not a bad idea, how about this? -> _"If you're requesting a new feature, tell us why you'd like this feature."_

Comment (phildini) [inline .github/issue_template.md]: Perfect!

Comment (glasnt) [inline .github/issue_template.md]: @shortdiv if you update this PR with these changes, I'll mergify it :sparkles:

Comment (shortdiv) [inline .github/issue_template.md]: @glasnt Just pushed up an update, I did a squash & rebase to reduce commit history on this one. Thanks!

**Pull request merged**